portfolio/src/main/webapp/index2.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Vincent Stevenson</title>
+        <link rel="stylesheet" href="style2.css">
+        <script src="script.js"></script>
```

Comment:
It looks like script.js isn't being used from this page -- if so, the script tag importing it should be removed